charts/sync/templates/sync.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: canaries.flanksource.com/v1
+kind: Topology
+metadata:
+  name: {{ .Values.topologyName }}
+spec:
+  icon: sync
+  type: Topology
+  schedule: "@every 2m"
+  components:
+    - name: Check Statuses
+      icon: helm
+      properties:
+        - name: Backlog
+          lookup:
+            postgres:
+              - query: SELECT count(*) FROM check_statuses where is_pushed = false
+                connection: connection://postgres/mission-control/mission-control 
+                display:
+                  expr: results.rows[0].count 
+        - name: Last push time
+          lookup:
+            postgres:
+              - query: SELECT max(time) FROM check_statuses where is_pushed = true
+                connection: connection://postgres/mission-control/mission-control 
+                display:
+                  expr: results.rows[0].max
+      checks:
+        - inline:
+            postgres:
+              - query: SELECT count(*) FROM check_statuses where is_pushed = false
```

Comment:
Shouldn't this be where min(time) > 1h (Need to make sure that the query uses indexes)